roles/firewall/test/tasks/main.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 
-- name: add port 5672
+- name: add port {{ rabbitmq_port }}
```

Comment:
This is role only exists to test the firewall role.  It doesn't need to be using these variables.  Let's remove this change from the pr.